heartbeat/monitors/util.go

```diff
@@ -35,12 +37,16 @@ const (
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function MakeJob should have comment or be unexported